lib/scope.js

```diff
@@ -279,7 +279,7 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
             if (node instanceof AST_SymbolCatch) {
                 var name = node.name;
                 var refs = node.thedef.references;
-                var scope = node.thedef.scope.parent_scope;
+                var scope = node.thedef.scope.parent_scope.parent_scope;
```

Comment:
As discovered in https://github.com/mishoo/UglifyJS2/pull/1604#issuecomment-287220620, `harmony` decided to wrap `AST_SymbolCatch` inside _two_ `AST_Scope`, one for `AST_Catch` and one for `AST_Try`.

Without trying to figure out whether that makes sense or not, just monkey-patch this to match behaviour on `master`.